ksql-engine/src/main/java/io/confluent/ksql/query/KafkaStreamsBuilder.java

```diff
@@ -15,10 +15,27 @@
 
 package io.confluent.ksql.query;
 
+import static java.util.Objects.requireNonNull;
+
 import java.util.Map;
 import org.apache.kafka.streams.KafkaStreams;
 import org.apache.kafka.streams.StreamsBuilder;
+import org.apache.kafka.streams.Topology;
 
 public interface KafkaStreamsBuilder {
-  KafkaStreams buildKafkaStreams(StreamsBuilder builder, Map<String, Object> conf);
+  BuildResult buildKafkaStreams(StreamsBuilder builder, Map<String, Object> conf);
+
+  class BuildResult {
```

Comment:
Added this class to avoid the caller having to also build the topology, (which was happening in most places)